src/Compute/Compute/Generated/Snapshot/Config/NewAzureRmSnapshotConfigCommand.cs

```diff
@@ -192,6 +192,13 @@ public partial class NewAzureRmSnapshotConfigCommand : Microsoft.Azure.Commands.
             HelpMessage = "Required if createOption is CopyFromSanSnapshot. This is the ARM id of the source elastic san volume snapshot.")]
         public string ElasticSanResourceId { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "When createOption CopyStart, the snapshot will be copied at a quicker speed. Possible values include: 'None', 'Enhanced'")]
+        [PSArgumentCompleter("Enhanced", "None")]
+        public string TierOption { get; set; }
```

Comment:
IS this supposed to be here? IS there a test for this new feature?

Reply:
yeah this seems new parameter to a cmdlet

Reply:
The parameter was requested a while back and seemed to have been blocked by SDK release. Added that in officially as part of this PR.